modules/system/models/MailTemplate.php

```diff
@@ -60,6 +60,23 @@ public static function listAllTemplates()
     }
 
     /**
+     * Returns a list of all mail templates.
+     * @return array Returns an array of the MailTemplate objects.
+     */
+    public static function allTemplates()
+    {
+        $result = [];
+        $codes = array_keys(self::listAllTemplates());
+
+        foreach ($codes as $code) {
+
```

Comment:
What's with the extra whitespace here?

Reply:
Line 72 can be removed. Thanks for the branch move!

Reply:
It's easier for me if you do that yourself on your branch so that I can merge it in via the PR.

Reply:
Done!